Lib/test/test_ctypes/test_dlerror.py

```diff
@@ -58,11 +58,14 @@ def test_null_dlsym(self):
         import subprocess
         import tempfile
 
-        retcode = subprocess.call(["gcc", "--version"],
-                                  stdout=subprocess.DEVNULL,
-                                  stderr=subprocess.DEVNULL)
-        if retcode != 0:
+        try:
+            retcode = subprocess.call(["gcc", "--version"],
+                                      stdout=subprocess.DEVNULL,
+                                      stderr=subprocess.DEVNULL)
+        except FileNotFoundError:
             self.skipTest("gcc is missing")
+        if retcode != 0:
+            self.skipTest("gcc is not working")
```

Comment:
```suggestion
            self.skipTest("gcc --version failed")
```

Reply:
rebased to latest main and applied this suggestion.